modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -485,6 +494,12 @@ private CompletableFuture<Void> createZonePartitionReplicationNode(
                                 stablePeersAndLearners,
                                 raftGroupListener,
                                 raftGroupEventsListener,
+                                // TODO: IGNITE-24371 - pass real isVolatile flag
+                                false,
+                                () -> zoneResourcesManager.getOrCreatePartitionStorage(
```

Comment:
Since now we do create and close resources on zone create and node stop let's also destroy them on zone drop.

Reply:
I think it should be implemented in https://issues.apache.org/jira/browse/IGNITE-24345 (or close to it)